src/pages/flash.astro

```diff
@@ -0,0 +1,191 @@
+---
+import '../styles/styles.css';
+import Button from '../components/Button.astro';
+---
+
+<head>
+    <title>Flashing Numbers</title>
+    <style>
+      body {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 100vh;
+        margin: 0;
+        font-family: Arial, sans-serif;
+        background-image: url('/mathArenaBkg.jpg');
```

Comment:
I need to make a PR down the line moving all this stuff into a common stylesheet. Out of scope for this PR